packages/frontend/editor-ui/index.html

```diff
@@ -5,9 +5,10 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width,initial-scale=1.0" />
 		<link rel="icon" href="/favicon.ico" />
-        <link rel="stylesheet" href="/static/prefers-color-scheme.css">
 		%CONFIG_SCRIPT%
-		<script src="/static/posthog.init.js" type="text/javascript"></script>
+		<link rel="stylesheet" href="/{{BASE_PATH}}/static/prefers-color-scheme.css">
+		<script src="/{{BASE_PATH}}/static/base-path.js" type="text/javascript"></script>
+		<script src="/{{BASE_PATH}}/static/posthog.init.js" type="text/javascript"></script>
```

Comment:
Using {{}} variables directly in index.html means the errors in dev console will be back – the added complexity by transforms in Vite config won't pay off at this point. For consistency and clarity, I'd propose adding back all scripts in index.html and process these variables with Vite [at dev time only](https://vite.dev/guide/using-plugins.html#conditional-application).

https://github.com/n8n-io/n8n/compare/account-for-subpath-when-serving-config.js...refactor-dev-mode-path-substitution